scripts/tampermonkey-alternative-method.js

```diff
@@ -20,7 +20,7 @@ function onQuizCreate() {
     if (isQuizCreated) return // Quick fix to ensure script is only loaded once
     isQuizCreated = true
 
-    // Start the magic
+    Start the magic
```

Comment:
That won't work, comments start with `//` and you just removed it.

Reply:
lmao